proprietary/design-tokens/src/component/utrecht/link-social.tokens.json

```diff
@@ -0,0 +1,15 @@
+{
+  "utrecht": {
+    "link-social": {
+      "background-color": { "value": "{utrecht.color.red.40}" },
+      "border-width": { "value": "2px" },
+      "border-color": { "value": "{utrecht.color.white}" },
+      "size": { "value": "40px" },
+      "distanced": { "value": "9.6px" },
+      "icon": {
+        "color": { "value": "{utrecht.color.white}" },
```

Comment:
How about we start with `link-social-color` and use that for the icon, and wait with introducing `link-social-icon-color` specifically when we need different colors for regular content and icons?